eq-author/src/components/withEntityEditor/index.js

```diff
@@ -68,23 +68,27 @@ const withEntityEditor = (entityPropName, fragment) => WrappedComponent => {
       }
     };
 
-    handleUpdate = () => {
-      if (!this.dirtyField) {
-        return;
-      }
+    handleUpdate = debounce(
```

Comment:
Did you explore any alternative solutions to preventing the request being triggered twice? Debouncing and this arbitrary 20ms time period seems like it could cause trouble in the future.